### PR TITLE
[new exo-run - 5] update dockerfiles with startup commands

### PR DESCRIPTION
**Pull request by cheahuychou**

### CODE REVIEW

File: exo-go/src/app_setup/index_test.go [hunk 1]
```diff
@@ -49,98 +52,144 @@ var _ = Describe("Setup", func() {
...
-	It("should include external dependencies as dependencies", func() {
-		exists := util.DoesStringArrayContain(dockerCompose.Services["todo-service"].DependsOn, "mongo3.4.0")
-		Expect(exists).To(Equal(true))
+	var _ = Describe("Dockerfile", func() {
+		It("should update the dockerfiles of internal services with the commands defined in service.yml", func() {
+			for _, serviceName := range internalServices {
+				actualBytes, err := ioutil.ReadFile(path.Join(appDir, serviceName, "Dockerfile"))
+				Expect(err).NotTo(HaveOccurred())
+				expected := `RUN yarn install`
+				Expect(string(actualBytes)).To(ContainSubstring(expected))
```
Comment (charlierudolph) [inline exo-go/src/app_setup/index_test.go]: Can we use `ToEqual`? I think this test may be giving a false positive as `RUN yarn install --production` already exists in the dockerfile.

Comment (cheahuychou) [reply exo-go/src/app_setup/index_test.go]: comment addressed

File: exo-go/src/app_setup/index_test.go [hunk 2]
```diff
@@ -37,115 +40,154 @@ var _ = Describe("Setup", func() {
...
+			Expect(dockerCompose.Services[serviceName].Ports).To(Equal(ports))
+			Expect(len(dockerCompose.Services[serviceName].Volumes)).NotTo(Equal(0))
+		})
 	})
 
-	It("should include external dependencies as dependencies", func() {
-		exists := util.DoesStringArrayContain(dockerCompose.Services["todo-service"].DependsOn, "mongo3.4.0")
-		Expect(exists).To(Equal(true))
+	var _ = Describe("Dockerfile", func() {
+		It("should update the dockerfiles of internal services with the commands defined in service.yml and not modify the commands in the original dockerfiles", func() {
```
Comment (charlierudolph) [inline exo-go/src/app_setup/index_test.go:157]: I can see where you are testing "should update the dockerfiles of internal services with the commands defined in service.yml" but I don't see anything testing "not modify the commands in the original dockerfile"

File: exo-go/src/app_setup/index_test.go [hunk 3]
```diff
@@ -37,115 +40,154 @@ var _ = Describe("Setup", func() {
...
+		setup, err = appSetup.NewAppSetup(appConfig, logger.NewLogger([]string{}, []string{}), appDir, homeDir)
 		Expect(err).NotTo(HaveOccurred())
 		err = setup.Setup()
 		Expect(err).NotTo(HaveOccurred())
-		expectedDockerComposePath := path.Join(appDir, "tmp", "docker-compose.yml")
-		Expect(osHelpers.FileExists(expectedDockerComposePath)).To(Equal(true))
-		dockerCompose, err = dockerHelpers.GetDockerCompose(expectedDockerComposePath)
+		expectedDockerComposeLocation := path.Join(appDir, "tmp", "docker-compose.yml")
+		Expect(osHelpers.FileExists(expectedDockerComposeLocation)).To(Equal(true))
+		dockerCompose, err = dockerHelpers.GetDockerCompose(expectedDockerComposeLocation)
```
Comment (charlierudolph) [inline exo-go/src/app_setup/index_test.go]: Can this be reverted back to expectedDockerComposePath?

**Comment by charlierudolph** [comment]
This is currently blocked on discussion about the future of the setup command in the `service.yml` as hopefully we can remove it.

**Comment by charlierudolph** [comment]
Sorry, but going to close this. We are going to remove the functionality for the startup command from the service.yml for now.

**Pull request closed without merging**